test/hlx_project_test.js

```diff
@@ -115,6 +115,31 @@ describe('Helix Project', () => {
       .catch(done);
   });
 
+  it('Logs to custom logger', (done) => {
```

Comment:
I don't understand how this works. the very first time the `counter` is invoked, i.e. with the first log, it will invoke `done()`. so why do you need the rest in the `then` ?